_targets.R

```diff
@@ -0,0 +1,40 @@
+library(targets)
+src_files <- Sys.glob('*/src/*.R')
+for (f in src_files) {
+  print(paste("sourcing", f))
+  source(f)
+}
+tar_option_set(packages = c("miceadds", "readr", "stringr",
```

Comment:
Are you actually using miceadds here? I'm not familiar with that package.

Reply:
Oops, that was leftover from old code. Removed now!